scripts/solc-bump.js

```diff
@@ -13,12 +13,14 @@ const PACKAGES_DIR = path.resolve(__dirname, "..", "packages");
  **/
 function updatePackages(dryRun = false) {
   const packages = fs.readdirSync(PACKAGES_DIR);
+  const packageJsonName = "package.json";
+  const encoding = "utf8";
   return packages
     .map(packageName => {
       // parse the package.json file for `packageName`
       const packagePath = path.resolve(PACKAGES_DIR, packageName);
-      const packageJsonPath = path.resolve(packagePath, "package.json");
-      const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, "utf8"));
+      const packageJsonPath = path.resolve(packagePath, packageJsonName);
+      const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, encoding));
```

Comment:
My opinion for this one is similar; there's no reason to write `encoding` rather than `"utf8"`.  Again, everyone knows what `"utf8" means; there's no magic number problem here.  Putting it in a constant called `encoding` once again just makes it look like there might be other encodings we have to deal with; but in fact, we never use any encoding but `"utf8"`.  As with the previous one, I think this change decreases clarity, not increases it.